libraries/bot-dialogs/src/main/java/com/microsoft/recognizers/text/datetime/resources/EnglishDateTime.java

```diff
@@ -1231,8 +1231,11 @@
 
     public static final ImmutableMap<String, Integer> DayOfMonth = ImmutableMap.<String, Integer>builder()
         .put("1st", 1)
+        .put("1th", 1)
```

Comment:
This is strange but I assume these additions (..th) are for a reason, right?

Reply:
I can't tell about this as it's a copy paste from RT. However, the PR which includes this in RT justify this as: `Support to recognize date like "1th", "2th" in all platforms.`